### PR TITLE
Updated info about GitPOAPs

### PR DESCRIPTION
**Pull request by bskrksyp9**
Added info on GitPOAP and it's procedure to claim. Source as per official discord server.

### CODE REVIEW

File: src/content/contributing/index.md [hunk 1]
```diff
@@ -83,6 +83,19 @@ We have low barrier to entry issues on our GitHub repository specifically design
...
 
+{For GitPOAP} (one time contribution)
+
+GitPOAP will automatically recognizes your merged contribution and lets you mint a unique contributors POAP on their platform itself!
+
+### How to claim {#how-to-claim}
+
+1. Visit [GitPOAP](https://www.gitpoap.io). 
+2. Connect with your wallet or even with your email id through sign in option.
+3. Search for your GitHub username, ETH address, ENS names or any GitPOAP to check if you're eligibile.
```
Comment (clauBv23) [inline src/content/contributing/index.md:94]:
```suggestion
3. Search for your GitHub username, ETH address, ENS names or any GitPOAP to check if you're eligible.
```

Comment (bskrksyp9) [reply src/content/contributing/index.md:94]: thanks for correction 👍🏼

**Review by wackerow — CHANGES_REQUESTED** [review]
Hey @bskrksyp9... I see two PRs for this, #10328 and this one... can you help clarify which you're looking to use by closing the other? Assuming you wanted to go with this one since it came after and has more details.

Few notes on this one, I would put the information about GitPOAP after our note about "If your contribution gets merged...", and also we'll need to clean up the `{}` usage... would use bolded text or headers as needed.

**Comment by bskrksyp9** [comment]
Hello @wackerow !

Yes you could modify the info as you mentioned, but the 2 PRs are different, though the change in content is same

One appears on Contributing page of site, where the other is from the github Readme Page

**Review by wackerow — APPROVED** [review]
@bskrksyp9 Just pushed a patch to this... shifted it below our standard POAP explainer, removed what looked like an unfinished note about tiered POAPs which we should leave off. Bringing it in, thanks!

**Pull request merged**